FFXCutsceneRemover/Resources/Transitions.cs

```diff
@@ -88,7 +90,9 @@ static class Transitions
                                            // Tidus is gone
                                            // Tidus gets hit by scales
                                            // Post Echuilles
-            { new GameState { RoomNumber = 282, Storyline = 285 }, new Transition { RoomNumber = 43, Storyline = 292, ForceLoad = true } },    // Kilika is destroyed
+            //{ new GameState { RoomNumber = 282, Storyline = 285 }, new Transition { RoomNumber = 220, Storyline = 285, ForceLoad = true } },    // Kilika FMV - Party members added back on reward screen - Bug: Need to enable menu to fix
+            { new GameState { RoomNumber = 220, Storyline = 287 }, new Transition { RoomNumber = 139, Storyline = 290, ForceLoad = true } },    // Recovering on the boat
```

Comment:
Just a reminder that ForceLoad = True by default in transitions. You only need to set it to False if you don't want a transition.